eemeter/caltrack/hourly.py

```diff
@@ -217,6 +249,27 @@ def json(self):
         )
         return data
 
+    @classmethod
+    def from_json(cls, data):
+        """ Loads a JSON-serializable representation into the model state.
+
+        The input of this function is a dict which can be the result
+        of :any:`json.loads`.
+        """
+
+        segment_models = [
+            CalTRACKSegmentModel.from_json(s)
+            for s in data.get('segment_models')
+        ]
+
+        c = cls(
+            segment_models,
+            pd.read_json(data.get('occupancy_lookup'), orient="split"),
```

Comment:
Instead of this line, I would suggest doing the following

```
occupancy_lookup = pd.read_json(data.get('occupancy_lookup'), orient='split')
occupancy_lookup.index = occupancy_lookup.index.astype(pd.Categorical(range(0,168)))

c = cls(
            segment_models,
            occupancy_lookup,
            pd.read_json(data.get('temperature_bins'), orient="split")
            )
```